### DISCUSSION TITLE
Workplace Search file link question

### DISCUSSION THREAD

**Discussion by paksak** [dadoonet/fscrawler#1528] [Q&A] [answered]
Hi, David.
As you suggeted, I ran python http.server  and I could run binary files on local env.
After that,  I made Workplace Search with fscrawler on procution system and I could't run with python http.server
cause python http.server cannot support non-local system.
What should I do for runnig files on production system?
And I set 'workplac_search server' as "http://192.168.150.150:3002" in fscrawler config, but indexed file url is always
set to '127.0.0.1' .
I think url should be "http://192.168.150.150:3002" and then file could be run.
I wonder this url would be valid with web sever on production system. 
Did I do something wrong?

**Comment by paksak** [comment]
I missed someting, so I added to my configuration as below
"workplace_search:
  url_prefix: "http://192.168.150.140/"
,and then file url is changed to 'http://192.168.150.140'.
But file doesn't open yet.
Is it python http.server problem? should I install Apache web server ?

**Reply by dadoonet** [reply]
I don't know and this is not under fscrawler control. Something you need to fix on your end.

**Comment by paksak** [comment]
I see, david.
I finally installed apache web server and upoloaded file on web server root directoy, then I could open files.
As far as I know, python http,server doesn't support production environement.
So if you want open file on prod. system, you'd better install apache web server.

**Reply by dadoonet** [reply]
Great! Do you think you could share that within a pull request to add this to the documentation?

**Reply by paksak** [reply]
Sure. I'd love to.
Have a good day!

**Comment by paksak** [comment, marked as answer]
After testing http.server again,It did work on production env.
It was firewall problem. My company blocked sever's 80 port and I did't know that.
I'm sorry for bothering you.
